client/src/Components/Navbar.js

```diff
@@ -45,7 +45,7 @@ const Navbar = (props) => {
     history.push(route);
     if (route === '/signup') {
       dispatch({ type: LOGOUT });
-      localStorage.clear();
+      fetch('/users/logout', { method: 'POST', credentials: 'same-origin' });
```

Comment:
should we also make this a request using axios and in the apiFunctions.js file

Reply:
Yes!  Definitely intended to have all async calls from from there. Missed this one (it was my victory fetch on the logout, think my excitement got in the way of the refactor 😃 ).